front/cypress/e2e/proposal_posting_anonymous.cy.ts

```diff
@@ -29,12 +24,12 @@ describe('Initiatives with anonymous participation allowed', () => {
   });
 
   it('admin can submit anonymous initiatives', () => {
-    const initiativeTitle = randomString(20);
-    const initiativeContent = randomString(60);
-
+    const initiativeTitle = randomString(10);
+    const initiativeContent = randomString(30);
```

Comment:
To speed up tests, it's important that we only have the minimum required length of content.